### ISSUE TITLE
WCAG 2.1 (Tues 6 May)

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-reviews-and-announcements#12]
## Document details

- **Last published document:** [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/)
- **Version/Type of document**: [Recommendation?]
- **W3C Staff Contact:** Kevin White
- **Working Group:** Accessibility Guidelines Working Group
- **Editor(s):** Alastair Campbell (Nomensa), Andrew Kirkpatrick (Adobe), Joshue O Connor (Invited Expert, InterAccess), Michael Cooper

## Review

- **Document: to review:** n/a

## Announcement

- **Announcement needed?:** Yes
- **Announcement channels:**
  - WAI email (public-wai-announce)
  - WAI news
  - W3C news
  - WAI LinkedIn - No?
- **Draft announcements blurbs provided in:**  [Google doc](https://docs.google.com/document/d/1xHNIG6QTpaZJq8hozYiGfE71oBZydIdxH_V7Ga8nNps/edit?usp=sharing)
- **Signatories on WAI email:** @@
- **Target date for announcement:**  2025-04-24
- **Date public feedback is due by:** n/a

### Key messages

Key messages for announcements:
- Announcement similar to one in December (but only for 2.1) - https://lists.w3.org/Archives/Public/public-wai-announce/2024OctDec/0002.html
- Publication is to fix minor technical issues that occurred with the December publication.

### Summary of changes since last publication

- n/a

### Review questions:
- n/a

## Related docs that need updating
- n/a

## Publication links:

- **Final published document:** https://www.w3.org/TR/2023/REC-WCAG21-20250424/
- **Links to announcements:** [link after review]
  - WAI email (public-wai-announce):  https://lists.w3.org/Archives/Public/public-wai-announce/2025AprJun/0000.html
  - WAI news: https://www.w3.org/WAI/news/2025-05-06/wcag-21/ 
  - W3C news: https://www.w3.org/news/2025/web-content-accessibility-guidelines-wcag-2-1-updated/

## Managing the review and publishing process

- Date the announcement wording needs to be ready for review by Team Contact: 22 April 2024

## Notes
...

**Comment by tamsinewing555** [comment]
@iadawn and @shawna-slh - here are the draft blurbs for the WCAG 2.1 announcement for you to review:
https://docs.google.com/document/d/1xHNIG6QTpaZJq8hozYiGfE71oBZydIdxH_V7Ga8nNps/edit?usp=sharing

Please can you help with the missing information and my questions.
